e2e/tests/booking.feature

```diff
@@ -35,6 +35,22 @@ Feature: Manage Temporary Accommodation - Booking
         And I attempt to mark the booking as arrived with required details missing
         Then I should see a list of the problems encountered marking the booking as arrived
 
+    Scenario: Extending a booking
```

Comment:
Respect for keeping the e2e tests up to date. We've got a bit lazy in this regard

Reply:
tbh it's causing some problems! I've accidentally pushed us into this odd situation where reference data comes from the API, *but* the UI still needs it's own copy of that data for the E2E tests

I was taking a peek at your E2E tests just now, and I saw you have 
```
  const lostBedReasonId = Cypress.env('lost_bed_reason_id') || throwMissingCypressEnvError('lost_bed_reason_id')
```

to get round this, but I don't think I want that everywhere we're selecting from a list of options, and I don't think I want to go back to my original approach of inspecting the page to find an ID for a well-known name. I'm wondering if I need to spend some time on a generic approach to essentially derive a set of reference data from a dropdown/radio list